source/type/pb_converter.cpp

```diff
@@ -1712,18 +1712,14 @@ void PbCreator::fill_fare_section(pbnavitia::Journey* pb_journey, const fare::re
 
         pbnavitia::Ticket* pb_ticket = nullptr;
         if (ticket.is_default_ticket()) {
-            if (!unknown_ticket) {
```

Comment:
So no more try to only build one unknown_ticket : looks OK, probably not so bad.
Any idea why that was like that? Just to shorten a bit the response?
The other possible choice of securing the lonely unknown-ticket but "keeping" the feature is too costly?

Reply:
I have no idea what was the functionnality behind this.
Since the bug has been present for 4 years https://github.com/CanalTP/navitia/commit/380eefab27a5b6e2c7f77a38629fc10c58c1632c I guess the functionnality is not relevant anymore